Classes/app.rb

```diff
@@ -79,7 +87,7 @@ def list_all_people
     end
   end
 
-  def create_book()
+  def create_book
```

Comment:
- [ ] in this method you are printing to the console, getting user input and saving the data, this is not in line with the single responsibility rule please have a look at [this article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid) for more details.